tests/sample_data/builds-running.json

```diff
@@ -3,17 +3,17 @@
         "test_treeherder": {
             "4b40022e5c4c": [
                 {
-                    "submitted_at": 1369231311,
-                    "buildername": "WINNT 5.2 profiling build",
+                    "submitted_at": 1376416924,
```

Comment:
changed to match a job that exists in the builds4hr sample data, so it will update it.